WebDriverAgentLib/FBSpringboardApplication.h

```diff
@@ -9,10 +9,12 @@
 
 #import <WebDriverAgentLib/FBApplication.h>
 
-@interface FBSpringboardApplication : FBApplication
-
 NS_ASSUME_NONNULL_BEGIN
 
+static NSString *const SPRINGBOARD_BUNDLE_ID = @"com.apple.springboard";
```

Comment:
Nice! However can you move it to '.m' file? This will complain about redefinition if you include this header twice.
If you need it outside, probably you don't, you can always add in header this
```
extern static NSString *const SPRINGBOARD_BUNDLE_ID;
```

Reply:
piece of cake :)